.gitlab/tests.yml

```diff
@@ -15,6 +15,26 @@ variables:
   stage: tests
   needs: [ build_base_venvs ]
   parallel: 4
+  services:
+    - name: datadog/agent:latest
+      alias: ddagent
+      variables:
+        DD_HOSTNAME: ddagent
+        DD_REMOTE_CONFIGURATION_ENABLED: true
+        DD_SITE: datadoghq.com
+        DD_API_KEY: invalid_but_this_is_fine
+        DD_REMOTE_CONFIGURATION_KEY: invalid_but_this_is_fine
+        DD_REMOTE_CONFIGURATION_REFRESH_INTERVAL: 5s
+        DD_DOGSTATSD_NON_LOCAL_TRAFFIC: true
+    - name: ghcr.io/datadog/dd-apm-test-agent/ddapm-test-agent:v1.17.0
```

Comment:
do we need an alias here too? or does `localhost` work for this?

Reply:
The tests pass as is. Other suites i'm testing with suggest that it does need an alias, but not certain as of now.